assets/prometheus-adapter/deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: prometheus-adapter
+  namespace: openshift-monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: prometheus-adapter
+  template:
+    metadata:
+      labels:
+        name: prometheus-adapter
+    spec:
+      containers:
+      - args:
+        - --cert-dir=/var/run/serving-cert
+        - --config=/etc/adapter/config.yaml
+        - --logtostderr=true
+        - --metrics-relist-interval=1m
+        - --prometheus-url=https://prometheus-k8s.openshift-monitoring.svc:9091
+        - --secure-port=6443
+        - --prometheus-ca-file=/etc/ssl/certs/service-ca.crt
+        image: quay.io/metalmatze/k8s-prometheus-adapter-amd64:b1b76b6159eb
+        name: prometheus-adapter
+        ports:
+        - containerPort: 6443
+        volumeMounts:
+        - mountPath: /tmp
+          name: tmpfs
+          readOnly: false
+        - mountPath: /var/run/serving-cert
+          name: volume-serving-cert
+          readOnly: false
+        - mountPath: /etc/adapter
+          name: config
+          readOnly: false
+        - mountPath: /etc/ssl/certs
+          name: serving-certs-ca-bundle
+          readOnly: false
+      priorityClassName: system-cluster-critical
+      securityContext: {}
+      serviceAccountName: prometheus-adapter
+      volumes:
+      - emptyDir: {}
+        name: tmpfs
+      - emptyDir: {}
+        name: volume-serving-cert
+      - configMap:
+          name: adapter-config
+        name: config
+      - configMap:
+          name: prometheus-serving-certs-ca-bundle
```

Comment:
this was just renamed to just `serving-certs-ca-bundle` (without the `prometheus-` prefix) in #175